src/app/examples/card-example/card-example.component.html

```diff
@@ -6,5 +6,16 @@
     Lorem, ipsum dolor sit amet consectetur adipisicing elit. Mollitia facere molestias recusandae necessitatibus ab veniam repellendus doloremque culpa quam libero, est quo accusamus cumque, in quia itaque cupiditate ratione repellat!
   </div>
 
+  <kirby-card-footer></kirby-card-footer>
+</kirby-card>
+
+<kirby-card [hasPadding]="true">
```

Comment:
Just for the sake of the discussion, should this not be the other way around? That we default to having padding, but the user should actively choose not to have it?

Or is this done to keep backwards compatibility?

Paging @hafstad

Reply:
I agree :) Will reverse it.